at_follows_flutter/example/lib/screens/follows_screen.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:at_follows_flutter/screens/connections.dart';
+import 'package:at_follows_flutter_example/services/at_service.dart';
+import 'package:at_follows_flutter_example/services/notification_service.dart';
+import 'package:flutter/material.dart';
+import 'package:at_utils/at_logger.dart';
+import 'package:at_follows_flutter_example/utils/app_strings.dart';
+
+class NextScreen extends StatefulWidget {
+  @override
+  _NextScreen createState() => _NextScreen();
+}
+
+class _NextScreen extends State<NextScreen> {
+  // final _formKey = GlobalKey<FormState>();
```

Comment:
please remove commented lines in this file

Reply:
removed commented lines